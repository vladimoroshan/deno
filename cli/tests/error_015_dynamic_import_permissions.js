(async () => {
  await import("http://localhost:4545/tests/subdir/mod4.js");
})();
