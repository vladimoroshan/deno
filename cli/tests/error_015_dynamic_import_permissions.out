error: Uncaught TypeError: network access to "http://localhost:4545/tests/subdir/mod4.js", run again with the --allow-net flag
